downstream/modules/platform/proc-account-linking.adoc

```diff
@@ -5,19 +5,43 @@
 [id="account-linking_{context}"]
 = Linking your account
 
- [TEXT]
+{PlatformNameShort} 2.5 provides a centralized location for users, teams and organizations to access the platform's services and features. When you upgrade from a previous version of {PlatformNameShort}, your existing account is automatically migrated to a single platform account. However, if you have multiple component accounts, your accounts must be linked to use the centralized features of the platform.
+
+The first time you log in to {PlatformNameShort} 2.5, the platform searches through the existing services to locate a user account with the credentials you entered. When there is a match to an existing account, that account is registered and becomes centrally managed by the platform. Any subsequent component accounts in the system are orphaned. This can cause problems accessing the services in {PlatformNameShort}. 
+
+To address this problem, you can use the account linking procedure to authenticate from any of your existing component accounts and still be recognized by the platform. Linking accounts will associate existing component accounts with the same user profile. 
+
+[IMPORTANT]
+====
+{ControllerNameStart} OAuth applications on the platform UI are not supported for 2.4 to 2.5 migration.
+====
 
 .Prerequisites
 
 * You have completed the upgrade process and have a legacy {PlatformNameShort} account and credentials.
 
 .Procedure
+If you have completed the upgrade process and have a legacy {PlatformNameShort} subscription, follow the account linking procedure below to migrate your account to {PlatformNameShort} 2.5.
 
+. Navigate to the login page for {PlatformNameShort}. 
+. In the login modal, select either *I have an {ControllerName} account* or *I have an {HubName} account* based on the credentials you have. 
+. On the next screen, enter the legacy credentials for the component account you selected and click btn:[Log in].
+. If you have successfully linked your account, the next screen shows your username with a green checkmark beside it. If you have other legacy accounts that you want to link, enter those account credentials and click btn:[Link] to link them to your centralized {Gateway} account.
+. Click btn:[Submit] to complete linking your legacy accounts. 
+. After your accounts are linked, depending on your authentication method, you may be prompted to create a new username and password. These credentials will replace your legacy credentials for each component account. 
```

Comment:
Nit-picky thing with "may". IBM says, "Use only to indicate permission. Otherwise, use can to indicate ability, or use might to indicate possibility."
```suggestion
. After your accounts are linked, depending on your authentication method, you might
 be prompted to create a new username and password. These credentials will replace your legacy credentials for each component account. 
```

Reply:
updated, ty